### PR TITLE
Feature/create case

### PR DESCRIPTION
**Pull request by isambard217**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,19 @@
   "dependencies": {
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-scripts": "1.1.4"
+    "react-redux": "^5.0.7",
```
Comment (AlhaythamElhassan) [inline package.json]: are we using Redux?

File: package.json [hunk 2]
```diff
@@ -5,12 +5,19 @@
...
+    "react-scripts": "1.1.4",
+    "redux": "^4.0.0",
```
Comment (AlhaythamElhassan) [inline package.json]: redux?

File: src/components/App.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 
+// Components
+import CaseForm from 'components/CaseFormComponent/CaseForm';
 
 export default () => {
     return (
         <div>
             Im the app
```
Comment (AlhaythamElhassan) [inline src/components/App.js]: do we still need Im the app

File: src/components/CaseFormComponent/CaseForm.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+        this.setState({ case : event.target.value })
+    };
+
+    render() {
+
+        return (
+
+            <Form onSubmit={this.handleSubmit}>
+                <h4> Add a Case </h4>
```
Comment (AlhaythamElhassan) [inline src/components/CaseFormComponent/CaseForm.js]: why is the h4 header ?

**Comment by isambard217** [comment]
LGTM

File: src/components/CaseFormComponent/CaseForm.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Button, Checkbox, Form } from 'semantic-ui-react';
+
+class CaseForm extends Component {
+
+    state = { case : " " };
+
+    handleSubmit = event => {
```
Comment (AlhaythamElhassan) [inline src/components/CaseFormComponent/CaseForm.js]: handle submit should at least log a case object to the console

**Pull request closed without merging**